_includes/head.html

```diff
@@ -28,6 +28,7 @@
     })(Tc.$);
   </script>
   <link href="http://fonts.googleapis.com/css?family=Raleway:400,700,300" media="screen" rel="stylesheet" type="text/css" />
+  <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
```

Comment:
you seem to be added a bunch of axios files below, why? Either use the one from the content delivery network or store them locally.